clang/labs/lab1/vasilev_ivan/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+add_llvm_library(DeprecationPlugin MODULE Find_Deprication.cpp PLUGIN_TOOL clang)
```

```suggestion
add_llvm_library(DeprecationPlugin MODULE Find_Deprecation.cpp PLUGIN_TOOL clang)
```
Please fix this typo in all files names.